lib/app/utils/validation.rb

```diff
@@ -19,7 +19,7 @@ def self.get_resource(json, version)
     RESOURCES = { dstu2: {}, stu3: {}, r4: {} }
     VALUESETS = {}
 
-    VERSION_MAP = { '1.0.2' => :dstu2, '3.0.1' => :stu3, '4.0.0' => :r4 }.freeze
+    VERSION_MAP = { '1.0.2' => :dstu2, '3.0.1' => :stu3, '4.0.0' => :r4, '4.0.1' => :r4 }.freeze
```

Comment:
should we still be mapping '4.0.0' to `:r4` or just '4.0.1'?

Reply:
I think we should keep 4.0.0 since I'm keeping the resource files from 3.0.0 and 3.0.1 still in this PR. We can get rid of it later if necessary.